rotkehlchen/chain/ethereum/types.py

```diff
@@ -118,4 +118,5 @@
         'YEARN_VAULT_V2',
         'ZERION_ADAPTER',
         'ILK_REGISTRY',  # added in 1.27.1 data migration
+        'ENS_PUBLIC_RESOLVER_2',
```

Comment:
It is already there. No need to make a remote update since we have agreed that missing contracts/abis will be searched in the packaged globaldb and this abi already exists there.